pkg/operator/ceph/cluster/cluster.go

```diff
@@ -236,6 +236,13 @@ func (c *ClusterController) initializeCluster(cluster *cluster, clusterObj *ceph
 		}
 		cephUser = cluster.Info.ExternalCred.Username
 	} else {
+		// Populate ClusterInfo with the last value
+		var err error
+		cluster.mons.ClusterInfo, _, _, err = mon.LoadClusterInfo(c.context, c.namespacedName.Namespace)
```

Comment:
This will fail if it's a new cluster. I meant to point out that it is only available if the cluster was already created in a previous reconcile. So we would need to move this between lines 251-252 where we know the cluster has been sufficiently configured and before `c.configureCephMonitoring()` is called.

Reply:
I was worried about that. `c.configureCephMonitoring()` is called directly below though... :( 

I might need @leseb's help here since he probably understands the ins and outs a little better.